rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -95,26 +90,38 @@ def __init__(
 
             # run the neural network building code on remote workers
             ray.get([w.build.remote() for w in self._workers])
+            # use only 1 max in flight request per worker since training workers have to
+            # be synchronously executed.
+            self._worker_manager = FaultTolerantActorManager(
+                self._workers,
+                max_remote_requests_in_flight_per_actor=1,
+            )
+            self._in_queue = deque(maxlen=20)
```

Comment:
This should be configurable from algorithm config.

Reply:
done